dashboard/test/fixtures/script_level.yml

```diff
@@ -39653,28 +39653,6 @@ script_level_68_11305:
   seed_key: 
   activity_section_position: 12
   levels: level_11305
-script_level_68_557:
```

Comment:
@dmcavoy 

Hi Dani! I noticed you have edited this file a bunch and I was wondering if you could help me answer some questions with the change I'm making. 

I'm having trouble finding where this fixture is used. If I just remove this, are there any side effects? Am I going to break some test somewhere like eyes tests that don't run in drone? Is there anything I need to go clean up? (Like the scripts in s3? I noticed this points to a level with code) 

Thank you!!!

Reply:
Fixtures are used in our tests to be able to mimic our course structure without actually seeding all of it. I think its safe to remove as long as tests pass when you remove it.